src/libraries/System.Net.Http/src/System/Net/Http/HttpClientHandler.AnyMobile.InvokeNativeHandler.cs

```diff
@@ -11,6 +11,12 @@
 using System.Security.Authentication;
 using System.Security.Cryptography.X509Certificates;
 
+#if TARGET_ANDROID
+using GetHttpMessageHandlerReturnType = object;
```

Comment:
Why is Android different here? The handler is:

* https://github.com/dotnet/android/blob/0b8a6b3f5242b51335e748eda6940e2e683ab959/src/Mono.Android/Xamarin.Android.Net/AndroidMessageHandler.cs#L65

Should it also be `HttpMessageHandler`?

Reply:
Returning `object` might be intentional, but I don't see a commit mentioning it. It could allow apps to trim away `System.Net.Http.HttpMessageHandler` if unused.

Reply:
> It could allow apps to trim away System.Net.Http.HttpMessageHandler if unused.

I do not see why it would make a difference. The method below has `HttpMessageHandler` cast that will keep the type around.